src/tutorials/0005-regular-files-api/03.md

```diff
@@ -25,23 +25,34 @@ ipfs.add(data, [options])
 So if we had the `File` object for an adorable photo of a kitten in our browser, accessible to us via a variable `catPic`, and we wanted to add it to our IPFS node, we could pass it into the `add` method as `data` like so:
 
 ```javascript
-ipfs.add(catPic)
+const result = await ipfs.add(catPic)
 ```
 
-Note that the `add` method can accept either a single `File` or an array, just in case you have more than one adorable animal photo to add to the node:
+The value of the variable `result` is an object in the following format:
```

Comment:
Any reason the format of the result is being described before the 2nd input option?

Reply:
Oh, sorry. You're teaching the `add` method here which can only take one thing, and the `addAll` following which takes an array? Will `addAll` survive if you pass it an array with only one item in it but freak out if you pass it just one thing that's not in an array?